tests/override.targets

```diff
@@ -22,6 +22,15 @@
     </ItemGroup>
   </Target>
 
+  <!-- Remove duplicate runtime references that occur when referencing System.Private.CoreLib from and SDK style project -->
+  <Target Name="RemoveRuntimeReferences"
+          Condition="'$(ReferenceSystemPrivateCoreLib)' == 'true' and '$(UsingMicrosoftNETSdk)' == 'true'" 
+          AfterTargets="ResolveReferences">
+    <ItemGroup>
+      <ReferencePath Remove="@(ReferencePath)" Condition="$([System.String]::new('%(ReferencePath.HintPath)').EndsWith('System.Runtime.dll')) or $([System.String]::new('%(ReferencePath.HintPath)').EndsWith('System.Collections.dll'))" />
```

Comment:
How are these getting duplicated? Wouldn't it be better to try and eliminate the duplication instead of fixing it after the fact?